superset/connectors/sqla/models.py

```diff
@@ -977,6 +977,17 @@ def _get_top_groups(
         return or_(*groups)
 
     def query(self, query_obj: Dict) -> QueryResult:
+        filters = security_manager.get_row_level_security_filters(
+            self
+        )  # Self is the Table model
+        if len(filters) > 0:
+            where = query_obj["extras"]["where"]
+            if len(where) > 0:
+                where = "({}) AND ".format(where)
```

Comment:
Would be better to use the SQLA expression toolkit. `from sqlalchemy import and_`, `from sqlalchemy.sql import text`

Reply:
Swapped